ui/de-lib/src/main/java/org/iplantc/de/analysis/client/views/AnalysisColumnModel.java

```diff
@@ -108,22 +143,26 @@ public String getPath() {
     }
 
     @Override
-    public HandlerRegistration addAnalysisAppSelectedEventHandler(AnalysisAppSelectedEvent.AnalysisAppSelectedEventHandler handler) {
+    public HandlerRegistration
```

Comment:
Not sure if that style changes were intentional. Guess me switching to IDEA and importing styles changed formatting in quite few places.